script.js

```diff
@@ -0,0 +1,18 @@
+const containMenu = document.getElementById('container_menu');
+const hamburgerMenu = document.getElementById('hamburger_menu');
+const closeButton = document.getElementById('close_button');
+const navElement = document.querySelectorAll('.menu_mobile a');
+
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly ensure when any of the mobile menu links are clicked, the mobile menu disappears.
*Hint: Check the class `.menu_mobile`*, I don't think that class exists in the index.html file.

Reply:
As the previous reviewer requested, you `button` position is a little off. Kindly check your CSS and fix it.